pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -284,7 +284,11 @@ self: super: {
   hsbencher = dontCheck super.hsbencher;
   hsexif = dontCheck super.hsexif;
   hspec-server = dontCheck super.hspec-server;
-  HTF = dontCheck super.HTF;
+  HTF = overrideCabal super.HTF (orig: {
+    preBuild = ''
+      patchShebangs --build scripts
```

Comment:
Oh, and can you add a comment about why this is needed?  Or why it wasn't needed before?

It would be great if you could report this upstreamv as well and put a link to the upstream github issue here as a comment.

Reply:
Added a comment.

There's not really a bug upstream. They use bash scripts, we just need to fixup for the sandbox. /bin/sh wouldn't be appropriate since they use bash features.

Reply:
I was thinking they might be able to move to `#!/usr/bin/env bash`, which I guess we wouldn't have to fixup (and would be more portable to other distros)?

But this is no big deal.

Reply:
Last I checked /usr/bin/env isn't in the sandbox?

Reply:
Yeah, it is not.

Reply:
Ah, that makes sense, sorry for the noise here then!